charms/worker/k8s/src/charm.py

```diff
@@ -222,8 +237,7 @@ def get_cluster_name(self) -> str:
         """
         if self._stored.cluster_name == "":
             if self.lead_control_plane and self.api_manager.is_cluster_bootstrapped():
-                # TODO: replace with API call once available from the snap
-                self._stored.cluster_name = str(uuid.uuid4())
+                self._stored.cluster_name = self._generate_unique_cluster_name()
```

Comment:
You probably want to leave this comment here, no? I imagine if the snap has set a cluster name, you wouldn't want to override it?

Reply:
Well, i have removed this comment b/c these changes to the charm actually override the bootstrap config to set a unique cluster name [right here](https://github.com/canonical/k8s-operator/pull/162/files#diff-5ba1576e3daac57729756610a1f51b9706129e66f250d7a8557ea5666d0c490aR317-R319)

this will be the official name of the cluster